cookbooks/aws-parallelcluster-environment/test/controls/system_authentication_spec.rb

```diff
@@ -35,7 +35,7 @@
 
   services = %w(sssd)
 
-  if os_properties.redhat8?
+  if os_properties.redhat?
```

Comment:
How is this change related to AL2023?

Reply:
This change is not related to AL2023. I was a miss during redhat 9 support